litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx

```diff
@@ -104,12 +108,15 @@ const AcceptedInvitations: React.FC = () => {
                   </Typography>
                 </IconButton>
               </div>
-              <Typography className={classes.projectRole}>
+              <Typography
+                data-cy="roleInAcceptedProject"
+                className={classes.projectRole}
+              >
                 {t('settings.teamingTab.invitation.acceptedInvitation.role')}:{' '}
                 {project.currentUserProjectRole}
               </Typography>
               <div className={classes.buttonDiv}>
-                <div data-cy="LeaveProject">
+                <div data-cy="LeaveAcceptedProject">
```

Comment:
Camel case is being used to label data-cy in most places. This can be written in camelCase as well if that convention is followed.

Reply:
Yes thanks for finding this. made the changes!